<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gbDatabaseOptions</class>
 <widget class="QGroupBox" name="gbDatabaseOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="toolTip">
   <string>Save databases selection</string>
  </property>
  <property name="statusTip">
   <string>Save databases selection</string>
  </property>
  <property name="title">
   <string>Query Database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;i&gt;m/z&lt;/i&gt; parent Filter Tolerance</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Minimal Intensity</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="gbAnalogs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Search for analogs</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;i&gt;m/z&lt;/i&gt; Tolerance</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spinAnalogTolerance">
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinParentFilterTolerance">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>17</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="spinMZTolerance">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.020000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="spinMinScore">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.700000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Minimal Cosine Score Value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>&lt;i&gt;m/z&lt;/i&gt; Tolerance</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbPolarity">
     <property name="maxVisibleItems">
      <number>2</number>
     </property>
     <property name="maxCount">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Polarity</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinMinIntensity">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Minimum Matched Peaks</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinMinMatchedPeaks">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbPolarity</tabstop>
  <tabstop>spinMZTolerance</tabstop>
  <tabstop>spinMinIntensity</tabstop>
  <tabstop>spinParentFilterTolerance</tabstop>
  <tabstop>spinMinScore</tabstop>
  <tabstop>gbAnalogs</tabstop>
  <tabstop>spinAnalogTolerance</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
